e2e_tests/src/_utils/reporter.js

```diff
@@ -0,0 +1,14 @@
+const logger = require('./logger');
+
+class MyCustomReporter {
```

Comment:
Just an improvement suggestion with the naming, can we use something more meaningful like `CLIReporter`?